plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestMinimalFunctionality.java

```diff
@@ -184,6 +187,31 @@ public void testLimitForSegmentQueries()
         assertQueryFails("SELECT * FROM " + TOPIC_AND_TABLE, "Segment query returned.*");
     }
 
+    @Test
+    public void testAggregationPushdown()
```

Comment:
Can you update the test cases to use `io.trino.sql.query.QueryAssertions.QueryAssert#isFullyPushedDown` to verify that pushdown actually takes places?

This has the added benefit that it automatically verifies correctness, by comparing results with pushdown and without.

Reply:
Sure will do

Reply:
BTW Maybe  you can  copy `TestPostgreSqlConnectorTest.test*AggregationPushdown` methods into Pinot tests (of course, the relevant ones?)

This would give you `isFullyPushedDown` usage and many test cases out of the box

Reply:
Updated.